x-pack/plugins/security_solution/cypress/e2e/detection_alerts/detection_page_filters.cy.ts

```diff
@@ -316,9 +315,9 @@ describe('Detections : Page Filters', { testIsolation: false }, () => {
     const idx = 3;
     const { FILTER_FIELD_TYPE, FIELD_TYPES } = FILTER_GROUP_EDIT_CONTROL_PANEL_ITEMS;
     editFilterGroupControls();
-    cy.get(CONTROL_FRAME_TITLE).eq(idx).trigger('mouseover');
-    cy.get(FILTER_GROUP_CONTROL_ACTION_EDIT(idx)).trigger('click', { force: true });
-    cy.get(FILTER_FIELD_TYPE).should('be.visible').trigger('click');
+    cy.get(CONTROL_FRAME_TITLE).eq(idx).realHover();
```

Comment:
as far as I checked we are not running Firefox on CI, in the future if we decide to enable in then can introduce a custom command that would pick a proper method per browser-basis
in terms of `realHover()`, yes, in my opinion it improves the test, because it mimics a real user behavior, instead of just triggering `eventListener` on `mouseover` on the element
to me, we should restrict using `trigger()` as it may result in unexpected behavior, both in tests, as well as in real user behavior

Reply:
I was under the impression that we do have the ability to run our Cypress tests against firefox. Here are some related files:

* https://github.com/elastic/kibana/blob/main/test/scripts/jenkins_security_solution_cypress_firefox.sh
* https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/config.firefox.ts

Are there issues with simulating the hover via untrusted events?

If we do currently have the ability to run Cypress on Firefox, and we don't currently have any issue with simulating hovers, then it would seem that this change has a disadvantage without a strong advantage.

Reply:
agree that there is an ability, but it doesn't mean that we use it :) 
as far as I know, we are not using jenkins for our pipelines anymore

almost every change that was made in this PR was necessary to remove the flakiness, if the Team that owns these tests decides to bring back support to run tests on Firefox then they can revert that or find another solution, but I want to make sure that once the code is merged I'm 99% sure it's flaky-free

Reply:
I see what you mean. So you expect the change to using `cypress-real-events` to reduce flakiness?

Reply:
correct, https://docs.cypress.io/api/commands/hover